### ISSUE TITLE
Determine evaluation metrics for numeric

### ISSUE THREAD

**Issue by kaitlynhair** [destiny-evidence/data-extraction-evaluation-toolkit#196]
### Description

Currently we have accuracy score, which measures the number of cases where there is an exact match.

With numeric data, the inexactness of a match is clearly defined, i.e. the distance between 1 and 1.1 is much less than the distance between 1 and 1001. That distance matters a lot for data extraction!

Here we can add a few regression metrics from sklearn.

- `mean_absolute_error`: describes the average absolute error, e.g. on average, the number of participants extracted by the LLM is wrong by 1.5
- `mean_absolute_percentage_error`: describes the average percentage error. With the above, [100, 101] is treated as just as wrong as [1,2] as they are both wrong by 1 unit. MAPE captures that the first pair is wrong by 1% and the second by 100%.

These would be sensible first metrics. We can easily implement any additional metrics from sklearn. We may want to in time think of custom metrics involving getting p scores the wrong side of a threshold or something like that.

### Definition of Done

- [ ] Add MAE and MAPE to numeric metrics [5 minute job] 

### Dependencies

- #Issue
